content/sensu-go/5.16/files/agent.yml

```
@@ -36,7 +36,7 @@ log-level: "debug" # available log levels: panic, fatal, error, warn, info, debu
 # authentication configuration
 ##
 #user: "agent"
-#password: "P@ssw0rd!"
+#password: "AGENT_PASSWORD"
```

From https://github.com/sensu/sensu-go/blob/release/5.16/backend/seeds/seeds.go#L76-L79 I'm understanding that the agent default password remains intact for this release.

```suggestion
#password: "P@ssw0rd!"
```